server/.env.example

```diff
@@ -1,2 +1,4 @@
 DEBUG=false
 CORS=*
+provider=https://api.harmony.one
+ewsContract=0x600bA532a96e2fDEBEE1275eC832D6b009267D31
```

Comment:
note: current production contract is 0x067B394Cbd08D08e565f886DEFDE906A7E42FB93. I will add a comment that this is local debugging address to avoid confusion